app/controllers/phylo_trees_controller.rb

```diff
@@ -54,8 +55,9 @@ def index
 
     # Augment tree data with sample attributes, number of pipeline_runs and user name
     @phylo_trees = @phylo_trees.as_json
+    sample_details_by_tree_id = sample_details_by_tree_id()
```

Comment:
maybe rename the var sample_details_by_tree_id? usually the function call is implicit so using it in the block could be confusing later